sdk-springboot/src/main/java/io/dapr/springboot/DaprController.java

```diff
@@ -18,6 +19,7 @@
 import reactor.core.publisher.Mono;
 
 import java.io.IOException;
+import java.nio.charset.Charset;

```

Comment:
seems unused.

Reply:
Right you are. I wonder why the static analyzer didn't pick up on this.